app/lib/logic/rolling_updates.rb

```diff
@@ -118,12 +118,6 @@ def missing_config
         end
       end
 
-      class NewProviderDocumentation < Base
-        def missing_config
-          false
-        end
-      end
```

Comment:
Apparently this leftover was not used anymore. But what if old customers have this rolling update in their custom configuration? Would then their upgrade fail? Should we keep this forever or document that it needs to be removed before upgrade?

Reply:
You mean if they have `new_provider_documentation: true` in their `rolling_updates.yml`.

Well, they are not supposed to touch this configuration in the first place :)

Secondly, why would they do that? Just to load a broken Swagger v2 Active Docs for 3scale APIs? :)

And most importantly, the `rolling_updates.yml` can contain extra features, and if there is no code to process them, then they would be just ignored.

Reply:
hmm, I think it failed for me like that in the past, maybe don't remember. But it makes sense, nobody would have used v2.